simple_functorch.py

```diff
@@ -740,6 +764,10 @@ def new_fn(d, *args):
         r, saved = self.inner.custom_vjp(batchify(fwd_fn), batchify(bwd_fn), *args)
         return r, saved
 
+    def custom_vmap(self, fn, batch_rule, *args):
+        result = batch_rule(self.inner, *args)
+        return result
```

Comment:
Unlike custom_vjp, custom_vmap does not call custom_vmap on the inner dispatcher!

Reply:
how come. It doesn't seem to me like Batched(Batched()) wouldn't apply. Perhaps you are saying, it is impossible for a batching rule to recursively refer to itself?